tests/pytests/test_profile.py

```diff
@@ -150,44 +150,9 @@ def testProfileAggregate(env):
   env.assertEqual(actual_res[1][4], expected_res)
 
 def testProfileCursor(env):
-  env.skipOnCluster()
   conn = getConnectionByEnv(env)
-  env.cmd('FT.CONFIG', 'SET', '_PRINT_PROFILE_CLOCK', 'false')
-
   env.cmd('ft.create', 'idx', 'SCHEMA', 't', 'text')
-  conn.execute_command('hset', '1', 't', 'hello')
-  conn.execute_command('hset', '2', 't', 'world')
-
-  expected_res = [['Total profile time'],
-                  ['Parsing time'],
-                  ['Pipeline creation time'],
-                  ['Iterators profile', ['Type', 'WILDCARD', 'Counter', 2L]],
-                  ['Result processors profile',
-                    ['Type', 'Index', 'Counter', 2L],
-                    ['Type', 'Loader', 'Counter', 2L]]]
-
-  actual_res = conn.execute_command('ft.profile', 'idx', 'aggregate', 'query', '*',
-                                    'load', 1, '@t',
-                                    'WITHCURSOR', 'COUNT', 10)
-  env.assertEqual(actual_res[2], expected_res)
-
-  actual_res = conn.execute_command('ft.profile', 'idx', 'aggregate', 'query', '*',
-                                    'load', 1, '@t',
-                                    'WITHCURSOR', 'COUNT', 1)
-  # test initial result                                    
-  env.assertEqual(actual_res[0], [1L, ['t', 'hello']])
-  env.assertGreater(actual_res[1], 0)
-  env.assertEqual(actual_res[2], None)
-
-  # test second result
-  actual_res = conn.execute_command('ft.cursor', 'read', 'idx', actual_res[1])
-  env.assertEqual(actual_res[0], [1L, ['t', 'world']])
-  env.assertGreater(actual_res[1], 0)
-  env.assertEqual(actual_res[2], None)
-
-  # test final result with profile
-  actual_res = conn.execute_command('ft.cursor', 'read', 'idx', actual_res[1])
-  env.assertEqual(actual_res[2], expected_res)
+  env.expect('ft.profile', 'idx', 'aggregate', 'query', '*', 'WITHCURSOR').error().contains('FT.PROFILE does not support cursor')
```

Comment:
We do not allow "external" use of cursor with ft.profile - OK
How can we test "internal" use of cursor with ft.profile when `RS_COORDINATOR` is defined?

Reply:
On the coordinator, if it wasn't enabled, the profile tests would have failed.